3.7/release-notes-known-issues37.md

```diff
@@ -47,6 +47,15 @@ Hot Backup
 | **Date Added:** 2019-10-09 <br> **Component:** Hot Backup API / arangobackup <br> **Deployment Mode:** DC2DC <br> **Description:** Hot Backup functionality in Datacenter to Datacenter Replication setups is experimental and may not work. <br> **Affected Versions:** 3.5.x, 3.6.x, 3.7.x <br> **Fixed in Versions:** - <br> **Reference:** N/A |
 | **Date Added:** 2019-10-09 <br> **Component:** arangobackup <br> **Deployment Mode:** All <br> **Description:** The startup option `--operation` works as positional argument only, e.g. `arangobackup list`. The alternative syntax `arangobackup --operation list` is not accepted. <br> **Affected Versions:** 3.5.x, 3.6.x, 3.7.x <br> **Fixed in Versions:** - <br> **Reference:** N/A |
 
+Validation
+----------
+
+| Issue      |
+|------------|
+| **Date Added:** 2019-03-17 <br> **Component:** JSON Schema <br> **Deployment Mode:** All <br> **Description:** Validation schemas using `"additionalProperties": false` need to handle built-in attributes: _key, _id, _rev, _from, _to **Affected Versions:** 3.7.alpha2 <br> **Fixed in Versions:** devel <br> **Reference:** N/A |
+| **Date Added:** 2019-03-17 <br> **Component:** JSON Schema <br> **Deployment Mode:** All <br> **Description:** Remote schemas will not work for security reasons this will probably not fix **Affected Versions:** all <br> **Fixed in Versions:** - <br> **Reference:** N/A |
+| **Date Added:** 2019-03-17 <br> **Component:** JSON Schema <br> **Deployment Mode:** All <br> **Description:** There is not proper way to show the user why the validation failed. This is under investigation but very hard to solve for complex schemas. For example when using not and anyOf, this would result in trees of possible errors. For now users should fall back to tools like (https://www.jsonschemavalidator.net/) **Affected Versions:** all <br> **Fixed in Versions:** - <br> **Reference:** N/A |
```

Comment:
```suggestion
| **Date Added:** 2019-03-17 <br> **Component:** JSON Schema <br> **Deployment Mode:** All <br> **Description:** There is currently no proper way to show the user why the validation failed. This is under investigation but very hard to solve for complex schemas. For example when using not and anyOf, this would result in trees of possible errors. For now users should fall back to tools like (https://www.jsonschemavalidator.net/) **Affected Versions:** all <br> **Fixed in Versions:** - <br> **Reference:** N/A |
```